lib/src/rendering/custom_layout.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:math';
+
+import 'package:feature_discovery/src/widgets.dart';
+import 'package:flutter/rendering.dart';
+
+enum BackgroundContentLayout {
+  background,
+  content,
+}
+
+class BackgroundContentLayoutDelegate extends MultiChildLayoutDelegate {
+  final OverflowMode overflowMode;
+
+  final Offset contentPosition;
+
+  final Offset backgroundCenter;
+  final double backgroundRadius;
+
+  BackgroundContentLayoutDelegate({
+    this.overflowMode,
+    this.contentPosition,
+    this.backgroundCenter,
+    this.backgroundRadius,
```

Comment:
All of these should be `@required` too because we have to use them in our implementation!

Reply:
I think it's a waste, but I will add them anyway... This code cannot magically change for the user 😅

Reply:
It's not for the user, it's just that this is so much more convenient for debugging!